packages/manager/apps/web-office/src/pages/dashboard/users/ModalEditUsers.component.tsx

```diff
@@ -0,0 +1,258 @@
+        </OdsText>,
+        true,
+      );
+    },
+    onError: (error: ApiError) => {
+      addError(
+        <OdsText preset={ODS_TEXT_PRESET.paragraph}>
+          {t('dashboard_users_edit_message_error', {
+            error: error.response?.data?.message,
+          })}
+        </OdsText>,
+        true,
+      );
+    },
+    onSettled: () => {
+      queryClient.invalidateQueries({
+        queryKey: [
+          licencePrepaidName
+            ? getOfficeLicenseQueryKey(selectedServiceName)
+            : getOfficeUsersQueryKey(selectedServiceName),
+        ],
+      });
+      onClose();
+    },
+  });
+
+  const handleSaveClick = () => {
+    const { firstname, lastname, login } = getValues();
+    editUser({
+      firstName: firstname,
+      lastName: lastname,
+      activationEmail: `${login}@${userDetail.activationEmail.split('@')[1]}`,
+    });
+  };
+
+  const handleCancelClick = () => {
+    onClose();
+  };
+
+  return (
+    <form>
+      <Modal
+        isOpen
+        title={t('dashboard_users_edit_title')}
+        color={ODS_MODAL_COLOR.information}
+        onClose={onClose}
+        isDismissible
+        isLoading={isLoading}
+        secondaryButton={{
+          label: t('common:cta_cancel'),
+          action: handleCancelClick,
+        }}
+        primaryButton={{
+          testid: 'edit-btn',
+          variant: ODS_BUTTON_VARIANT.default,
+          label: t('common:cta_confirm'),
+          isDisabled: !isDirty || !isValid,
+          isLoading: isSending,
+          action: handleSaveClick,
+        }}
+      >
+        <>
+          <OdsText preset={ODS_TEXT_PRESET.paragraph} className="mt-5">
+            {t('dashboard_users_edit_mandatory_field')}
+          </OdsText>
+
+          <Controller
+            control={control}
+            name="firstname"
+            rules={{ required: true }}
+            render={({ field: { name, value, onBlur, onChange } }) => (
+              <OdsFormField
+                error={errors.firstname ? String(errors.firstname.message) : ''}
+                className="mt-5"
+              >
+                <label slot="label">
+                  {t('dashboard_users_edit_form_label_firstname')} *
+                </label>
+
+                <OdsInput
+                  type={ODS_INPUT_TYPE.text}
+                  name={name}
+                  value={value}
+                  data-testid="input-firstname"
+                  hasError={!!errors.firstname}
+                  onOdsBlur={onBlur}
+                  onOdsChange={onChange}
+                ></OdsInput>
+              </OdsFormField>
+            )}
+          />
+
+          <Controller
+            control={control}
+            name="lastname"
+            rules={{ required: true }}
+            render={({ field: { name, value, onBlur, onChange } }) => (
+              <OdsFormField
+                error={errors.lastname ? String(errors.lastname.message) : ''}
```

Comment:
could ` error={String(errors?.lastname?.message})` work ?

Reply:
It's fixed